docs/reference/settings/security-settings.asciidoc

```diff
@@ -1056,6 +1056,34 @@ Specifies the supported protocols for TLS/SSL.
 Specifies the
 cipher suites that should be supported.
 
+[float]
+[[ref-kerberos-settings]]
+===== Kerberos realm settings
+
+For a Kerberos realm, the `type` must be set to `kerberos`. In addition to the
+<<ref-realm-settings,settings that are valid for all realms>>, you can specify
+the following settings:
+
+`keytab.path`:: Specifies the path to the Kerberos keytab file that contains the 
+service principal used by this {es} node. It expects the keytab file to be present in `ES_PATH_CONF`
+and have read permissions. Required.
```

Comment:
Does this mean that the keytab file *must* be in the ES_PATH_CONF, or is that just the default location?

Reply:
Do we say something similar for other files such as certificates? I think we should just be consistent with what we say